docs/tutorials/nemo-rl-grpo/setup.md

```diff
@@ -0,0 +1,99 @@
+(training-nemo-rl-grpo-setup)=
+
+# Setup
+
+## Enter a GPU Node
+
+**Estimated Time:** ~5 minutes
+
+Launch an interactive Slurm session to run training commands. Refer to the [NeMo RL Cluster Setup documentation](https://docs.nvidia.com/nemo/rl/latest/cluster.html#interactive-launching) for more details.
+
+```bash
+NUM_ACTOR_NODES=1
+ACCOUNT=<ACCOUNT_NAME>
+JOB_NAME=<JOB_NAME>
+PARTITION=<PARTITION>
+
+# Use the official NeMo RL container from NGC
+# See: https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo-rl
+CONTAINER=nvcr.io/nvidia/nemo-rl:v0.4.0
+CONTAINER_WORKDIR=$PWD
+MOUNTS="$PWD:$PWD"
+srun \
+    --nodes=${NUM_ACTOR_NODES} \
+    --ntasks=1 \
+    --account=${ACCOUNT} \
+    --job-name=${JOB_NAME} \
+    --partition=${PARTITION} \
+    --time=04:00:00 \
+    --gres=gpu:8 \
+    --no-container-mount-home \
+    --container-name=nemo-gym \
+    --container-mounts="${MOUNTS}" \
+    --container-image="${CONTAINER}" \
+    --container-workdir=$CONTAINER_WORKDIR \
+    --pty /bin/bash
+```
+
+## Clone and Setup NeMo RL + NeMo Gym
+
+**Estimated Time:** ~15-20 minutes
+
+```bash
+# Clone NeMo RL repository
+git clone https://github.com/NVIDIA-NeMo/RL
+cd RL
+
+# Clone NeMo Gym as a submodule
+git clone https://github.com/NVIDIA-NeMo/Gym.git 3rdparty/Gym-workspace/Gym
+
+# Initialize all submodules (Megatron, AutoModel, etc.)
+git submodule update --init --recursive
+
```

Comment:
I see you've removed this bit. Without this, I observe issues with the Ray stashed in the container.

`rm -rf /opt/ray_venvs/*`